htmlBody/bin/htmlBody.dart

```diff
@@ -0,0 +1,14 @@
+
+import 'package:http/http.dart' as http;
+
+void main(List<String> arguments) async {
+  // This example uses the Google Books API to search for books about http.
+  // https://developers.google.com/books/docs/overview
+  var url = 'https://example.com';
+  // Await the http get response, then decode the json-formatted response.
+  var response = await http.get(url);
+  if (response.statusCode == 200) {
+    print(response.body);
```

Comment:
The point of exercise 6 is to print the title. Can you import `html`, then find the <title> and print its text? Your program should display "Example Domain".